### ISSUE TITLE
Memory Leak in aireplay-ng

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1150]
**Reported by Moshe Kaplan  me on 16 Jul 2013 05:46 UTC**

In `aireplay-ng.c:2839-2842`, memory is `malloc`ed, but never `free`d.
```
if ( opt.ringbuffer )
	arp = (struct ARP_req*) malloc( opt.ringbuffer * sizeof( struct ARP_req ) );
else
	arp = (struct ARP_req*) malloc( sizeof( struct ARP_req ) );
```

**Comment by aircrack-ng** [comment]
**Attachment by Moshe Kaplan  me on 16 Jul 2013 05:48 UTC**

Uploaded file [1150_patch.diff](https://github.com/aircrack-ng/trac-attachments/blob/master/1150/20130716-054810_744-1150_patch.diff)

**Comment by aircrack-ng** [comment]
**Comment by misterx on 16 Jul 2013 05:46 UTC**

Fixed in aircrack-ng/aircrack-ng@0fdf02b4e6a931bb6a8bd21baf3afcc801ed6c86
